index.html

```diff
@@ -123,28 +118,31 @@ <h4>
             />
           </a>
         </figure>
+      </div>
         <input type="radio" name="favourites" id="tab2">
-        <label for="favouritegames">
```

Comment:
The `for` attribute must have the same value is the `id` for whichever control it is to be associated with